index.html

```diff
@@ -22,7 +22,7 @@
       allow="fullscreen"
       id="next-player-01"
       name="next-player-01"
-      src="https://video.nest.com/embedded/live/Gmt1BYseCd?autoplay=1"
+      src="https://www.youtube.com/embed/CvOB-Is_yYU?autoplay=1&amp;origin=https://koti.possu.org/norppa.html&amp;mute=1"
```

Comment:
You could check the attributes above so this is consistent with the other frames. The other youtube frames don't have `allow`.